src/index.ts

```diff
@@ -1,56 +1,63 @@
 import xml2js from 'xml2js'
 
+type Maybe<T> = T | null
```

Comment:
I just found `parse()` can return null. However, does `TestSuites` and the `TestSuite` property can be null?
I think it is correct to make them optional types, but adding `Maybe` would make them different types than they really are.

If the XML does not have a certain key, then the returned object does not have the same key.
And even if the XML has a key with a value of null, the returned object will have a value of String "null". 

```js
  it('tests="null" is impossible case', async () => {
    const xml = `
    <?xml version="1.0" encoding="UTF-8"?>
    <testsuites>
      <testsuite failures="1" tests="null">
      </testsuite>
    </testsuites>
    `
    const parsed = await junit2json.parse(xml)

    expect(parsed).toEqual({
      testsuite: [
        { failures: 1, tests: "null" }, // It convert to String "null"
      ]
    })
  })
```

Can you add a test if you know some case returned values include null?